config/i18n.json

```diff
@@ -980,6 +980,8 @@
     "SortByAmmoType": "Ammo Type",
     "SortByAmount": "Stack Size",
     "SortByClassType": "Required Class",
+    "SortByCrafted": "Crafted",
+    "SortByDeepsight": "Deepsight",
```

Comment:
The other D2-specific sorting options have a "(D2)" suffix. Granted, it should be obvious those are D2 but maybe for consistency?

Reply:
fair!